gef-ui/server/controllers/supporting-information/supporting-documents/index.test.js

```diff
@@ -373,22 +373,6 @@ describe('controllers/supporting-documents', () => {
       });
     });
 
-    // it('returns error if error thrown when uploading', async () => {
```

Comment:
Can you please add a TO-DO to write a test coverage around this please.

Reply:
Done